### PR TITLE
Sidenav adjustments, including adding "Reference" section

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #975

Staged at https://angulardart-org-dev.firebaseapp.com

### CODE REVIEW

File: src/_includes/linklist.html [hunk 1]
```diff
@@ -1,24 +1,36 @@
-{% for link in linklist  %}
-  <h4 {% if page.url contains link.permalink %}class="active"{% endif %}>
-    {{ link.title }}
+{%- comment %}Strip out any trailing '/', etc. for exact URL match.{% endcomment -%}
```
Comment (kwalrath) [inline src/_includes/linklist.html]: I'm curious, why the added `-`s?

Comment (chalin) [reply src/_includes/linklist.html]: It strips the whitespace from before/after the Liquid directive.

**Comment by kwalrath** [comment]
Looks great, except for the differences between angular CSS and regular site CSS. E.g. compare the sidenavs of https://angulardart-org-dev.firebaseapp.com/tools and https://angulardart-org-dev.firebaseapp.com/api.

**Comment by chalin** [comment]
Yes, but a lot more work would be needed to harmonize the main site styles with the Angular specific styles. Possibly something for another day.

**Comment by kwalrath** [comment]
I've filed #980 about the style differences.

**Pull request merged**